### ISSUE TITLE
MegaBank Example With POJO Rules not working

### ISSUE THREAD

**Issue by abhineet27** [rulebook-rules/rulebook#65]
I don't think the MegaBank Example With POJO Rules are documented properly and even its working ?

I tried to simulated MegaBank Example With POJO Rules but the rules are not getting invoked.
Result is simply coming null.
Applicant qualified for the following rate: null null

**Comment by Clayton7510** [comment]
Should be. It was tested in prior releases. However, the example is not part of the unit tests. And there's always a possibility that I fat fingered something along the way in the README :)
I'll check it out tonight and add the example into the develop branch. I've been meaning to add some working examples into the project source for a while now, anyway.

**Comment by Clayton7510** [comment]
I just checked it out and there was some small errata that I have since corrected in the README example. I also updated the develop branch with the working megabank example in a gradle project. 

Finally, there was an issue that must have carried over from v0.5 where the result of chained rules is available to be updated but not read. So, stuff like result *= 4 wasn't working because result wasn't set. Weird problem. That took me a bit to diagnose. Anyway, it's fixed in 0.6.2-SNAPSHOT and in the develop branch. And I added a test to the unit test suite to ensure if that breaks again, it will be caught ;)

Thanks for raising this issue!
